### PR TITLE
Using same trick as before in the hope on pleasing travis' gods

### PR DESCRIPTION
**Pull request by lorenzo**
Pleasing the travis' gods

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -76,8 +76,7 @@ before_script:
   - composer install --prefer-dist --no-interaction
 
 script:
-  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION = 7.* ]]; then export CODECOVERAGE=1 ; phpdbg -qrr vendor/bin/phpunit --coverage-clover=clover.xml; fi
```
Comment (ADmad) [inline .travis.yml]: I have reported the problem to [codecov](https://github.com/codecov/support/issues/217) today and they are usually pretty quick to respond/resolve issues. So I would suggesting keep this to help debug the problem.

Comment (lorenzo) [inline .travis.yml]: We can keep a separate branch where we keep the line, so that they can debug until they fix the problem(s)

Comment (ADmad) [inline .travis.yml]: Okay

Comment (ADmad) [inline .travis.yml]: You should also remove the `after_success:` section which pushes the data to codecov.

Comment (markstory) [inline .travis.yml]: The coverage build is segfaulting which to me indicates an issue with xdebug and or php7.

**Comment by lorenzo** [comment]
Merging... Will submit another branch reverting the changes in travis.yaml shortly

**Pull request merged**